app.js

```diff
@@ -25,6 +26,8 @@ app.use(express.static(path.join(__dirname, "public")));
 
 // Routes
 app.use("/", blogRoutes);
+app.use("/", userRoutes);
```

Comment:
Does this cause a conflict when a user visits the `/` path?

Reply:
not right now as I can use routes and it is fine but I will consider this for later